experiment.py

```diff
@@ -76,8 +77,7 @@ def __init__(self, entry):
         for cell in self.cells.values():
             for label in labels:
                 assert label in cell.labels
-            for cre in cre_types:
-                assert cre in cell.labels
+            assert cell.cre_type in cre_types
```

Comment:
ah right, so that's why I had to change it because "unknown" is not a cre_type that has a label that would be in `cell.labels` and so that assertion failed for unknowns. We could put another qualifier in there that checks if it's an unknown and then makes sure that the cell isn't labeled as another cre_type?